packages/frontend/src/page/toolbar.tsx

```diff
@@ -33,13 +34,23 @@ const Brand = () => (
     </A>
 );
 
-/** Button that navigates to the root help page. */
+/** Button that navigates to the root help page or theory documentation page. */
 export function HelpButton() {
     const navigate = useNavigate();
-
-    return (
-        <IconButton onClick={() => navigate("/help")} tooltip="Get help about CatColab">
-            <CircleHelp />
-        </IconButton>
-    );
+    if (selectedTheory() == null) {
```

Comment:
Instead of using `selectedTheory()`, create a `TheoryHelpButton` that takes the theory as a prop. Then pass the theory as `props.liveModel.theory()` from inside the `ModelDocumentEditor`.